### DISCUSSION TITLE
QQ and Resend support.

### DISCUSSION THREAD

**Discussion by yenharvey** [logto-io/logto#6765] [General]
Why isn't there support for QQ login and the option to resend the verification email? Are there any plans to implement these features in the future?
<img width="1200" alt="image" src="https://github.com/user-attachments/assets/4ec6f878-3fcd-4d25-8cbd-df03d42ebc6c">
<img width="792" alt="image" src="https://github.com/user-attachments/assets/2c6315ca-40eb-446a-b2fe-c71d33617a22">

**Comment by darcyYe** [comment]
Hi @YenHarvey , would you like to post this as two separated feature requests in [issues](https://github.com/logto-io/logto/issues) so that we can better collect users feedbacks?

**Reply by yenharvey** [reply]
Got it👌.
